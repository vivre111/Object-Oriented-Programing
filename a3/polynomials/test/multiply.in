create a -9/2 8 4/3 4 2/1 2 0/5 1
create b 3/5 0
create c 0/2 12
create d 9/-2 12
multiply c d
print c
print d
multiply a b
multiply a c
multiply a d 
multiply b c
multiply a a
multiply b a

