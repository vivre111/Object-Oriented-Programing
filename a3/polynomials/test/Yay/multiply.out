0
0
(-9/2)x^12
(-27/10)x^8 + (4/5)x^4 + (6/5)x^2
0
(81/4)x^20 + (-6)x^16 + (-9)x^14
0
(81/4)x^16 + (-12)x^12 + (-18)x^10 + (16/9)x^8 + (16/3)x^6 + (4)x^4
(-27/10)x^8 + (4/5)x^4 + (6/5)x^2
